### ISSUE TITLE
Wrong class "form-control" for some select lists

### ISSUE THREAD

**Issue by gs71** [phpmyadmin/phpmyadmin#17728]
### Describe the bug

Some twig files in the templates folder have the wrong class "form-control" (instead of "form-select") applied to `select` elements.
The result is that these select lists are not properly displayed (wrong size, and down arrow is missing).

### To Reproduce

Look at broken select lists in SQL, Operations and Database pages.

### Expected behavior

Consistent interface.

### Server configuration

    Operating system: Linux
    Web server: Apache
    Database version: MariaDB 10.9.2
    PHP version: PHP 8.1.10
    phpMyAdmin version: 5.2.0 or 5.3.0-dev+20220917.b9499b51c8

### Client configuration

    Browser: any
    Operating system: any

### Additional context

Here the list of the files to fix (replace form-control with form-select):
```

grep -r \<select . | grep form-control 

templates/columns_definitions/column_definitions_form.twig:                  <select class="form-control" name="tbl_storage_engine" aria-label="{% trans 'Storage engine' %}">
templates/database/operations/index.twig:            <select class="form-control" lang="en" dir="ltr" name="db_collation" id="select_db_collation">
templates/sql/query.twig:                  <select class="form-control resize-vertical" id="fieldsSelect" name="dummy" size="{{ textarea_rows }}" ondblclick="Functions.insertValueQuery()" multiple>
templates/sql/query.twig:            <select class="form-control" name="id_bookmark" id="id_bookmark">
templates/table/operations/index.twig:            <select id="tableOrderFieldSelect" class="form-control" name="order_field" aria-describedby="tableOrderFieldSelectHelp">
templates/table/operations/index.twig:              <select id="moveTableDatabaseInput" class="form-control" name="target_db" aria-label="{% trans 'Database' %}">
templates/table/operations/index.twig:          <select class="form-control" name="new_tbl_storage_engine" id="newTableStorageEngineSelect">
templates/table/operations/index.twig:          <select class="form-control" id="collationSelect" lang="en" dir="ltr" name="tbl_collation">
templates/table/operations/index.twig:            <select class="form-control" name="new_pack_keys" id="new_pack_keys">
templates/table/operations/index.twig:            <select class="form-control" id="new_row_format" name="new_row_format">
templates/table/operations/index.twig:              <select class="form-control" name="target_db" aria-label="{% trans 'Database' %}">
templates/table/operations/index.twig:          <select class="form-control resize-vertical" id="partition_name" name="partition_name[]" multiple required>
```

**Comment by MikeRich88** [comment]
Would this issue be causing the select controls to be drawn without any indication that they are select controls when using the Original theme on Safari 15.6.1?

<img width="580" alt="image" src="https://user-images.githubusercontent.com/7113073/191327479-9bb49978-facc-43f5-92f2-8b6e35fde93a.png">

Look at ROW_FORMAT and Storage engine and stuff. Are those select menus? I have no idea, there's no indication that they are.

**Comment by gs71** [comment]
Exactly. this bug affects some select controls on any theme and any browser.

**Comment by sharma62** [comment]
I want to fix the issue.

**Comment by williamdes** [comment]
> I want to fix the issue.

@serjlc has made a fix for it in #17733
